.github/workflows/legacy-minicart-e2e-tests.yml

```
@@ -0,0 +1,340 @@
+                      if ( trigger === 'pre-release' ) {
+                        githubEvent = 'release-checks';
+                      }
+
+                      const child_process = require( 'node:child_process' );
+                      child_process.execSync( `pnpm utils ci-jobs ${ prNumberFilter } --event ${ githubEvent } --test-type e2e` );
+
+    project-test-jobs:
+        name: '${{ matrix.name }}'
+        runs-on: ubuntu-latest
+        needs: 'project-jobs'
+        if: ${{ !cancelled() && ( github.event_name != 'pull_request' || needs.identify-jobs-to-run.outputs.needs-code-validation == 'true' ) && needs.project-jobs.outputs.test-jobs != '[]' }}
```

actionlint (1.7.7)

property "identify-jobs-to-run" is not defined in object type {project-jobs: {outputs: {report-jobs: string; test-jobs: string}; result: string}}

(expression)